src/coreclr/nativeaot/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/CallConverterThunk.cs

```diff
@@ -119,10 +120,11 @@ private static unsafe bool CallConverterThunk_LazyCctor()
 #if TARGET_UNIX
             // TODO
 #else
-            CallingConventionConverter_GetStubs(out ReturnVoidReturnThunk, out ReturnIntegerPointReturnThunk, out CommonInputThunkStub
+            CallingConventionConverter_GetStubs(out ReturnVoidReturnThunk, out ReturnIntegerPointReturnThunk,
 #if CALLDESCR_FPARGREGSARERETURNREGS
+                                                out CommonInputThunkStub
 #else
```

Comment:
I do not see why this change is needed.

Reply:
This is purely to satisfy style cop. I may suppress it on these code. It did not like comma before other parameters.

Reply:
Suppressing the warning would be better. The ifdefs do not make logical sense with this change.